docs/src/pages/components/box/box-pt.md

```diff
@@ -4,30 +4,29 @@ title: Componente React Box
 
 # Box
 
-<p class="description">O componente Box serve como um componente wrapper (invólucro) para a maioria das necessidades de uso com CSS.</p>
+<p class="description">O componente Box serve como um componente encapsulador (wrapper) para a auxiliar na maioria das necessidades de uso com CSS.</p>
 
-O componente Box empacota [ todas as funções de estilo](/system/basics/#all-inclusive) que estão expostas em `@material-ui/system</ code>.
-Ele é criado usando a função <a href="/styles/api/#styled-style-function-component"><code>styled()`</a> do `@material-ui/styles`.
+O component Box compõe [todas as funções de estilo](/system/basics/#all-inclusive) que são expostas no `@material-ui/system`. Ele é criado usando a função [`styled()`](/styles/api/#styled-style-function-component) de `@material-ui/core/styles`.
 
 ## Exemplo
 
-[A paleta](/system/palette/) de funções de estilo.
+[A paleta](/system/palette/) com funções de estilo.
 
 ## Sobrescrevendo componentes do Material-UI
 
-O componente Box envolve seu componente. Ele cria um novo elemento DOM, uma `<div>` por padrão que pode ser modificada através da propriedade `component`. Digamos que você queira usar um `<span>`:
+O componente Box envolve seu componente. Ele cria um novo elemento DOM, uma `<div>` por padrão, comportamento que pode ser modificado através da propriedade `component`. Digamos que você queira usar um `<span>`:
 
 ```jsx
 <Box component="span" m={1}>
   <Button />
 </Box>
 ```
 
-Isso funciona muito bem quando as alterações precisam ser isoladas em um novo elemento DOM. Por exemplo, a você pode alterar as margens da mesma maneira.
+Isso funciona muito bem quando as alterações precisam ser isoladas em um novo elemento DOM. Note no exemplo, a forma que você alterou a margem.
 
-No entanto, às vezes, você precisa modificar o elemento DOM subjacente. Por exemplo, você deseja alterar a cor do texto do botão. O componente `Button` define sua própria cor. A herança por CSS não nos ajuda nesse caso. Para contornar o problema, você tem duas opções:
+No entanto, às vezes, você precisa modificar o elemento DOM subjacente. Por exemplo, você deseja alterar a cor do texto do botão. O componente `Button`, por sua vez, define sua própria cor. A herança por CSS não irá ajudar nesse caso. Para contornar o problema, você tem duas opções:
 
-1. Usar [`React.cloneElement()`](https://reactjs.org/docs/react-api.html#cloneelement)
+1. Usar [`React.cloneElement()`](https://pt-br.reactjs.org/docs/react-api.html#cloneelement)
```

Comment:
Interesting! @jaironalves, @DDDDDanica. I like how React keeps the same hash for the headers between locales. Material-UI doesn't, which makes language negotiation harder.

Reply:
Hi @oliviertassinari , 

If I understand the structure of their documentation well, they have a language repository, made in Gastby.
https://github.com/reactjs/reactjs.org
https://github.com/reactjs/pt-BR.reactjs.org

They synchronize via a bot between the main repository. The community then works to resolve conflicts and translations.
So, each one is a different publication. At least that's what I understood.

Reply:
@jaironalves Thanks for the link, I was comparing 

- https://reactjs.org/docs/react-api.html#cloneelement
- https://zh-hans.reactjs.org/docs/react-api.html#cloneelement

vs 

- https://material-ui.com/components/rating/#simple-ratings
- https://material-ui.com/zh/components/rating/#%E7%AE%80%E5%8D%95%E8%AF%84%E5%88%86

I believe we should have https://material-ui.com/zh/components/rating/#simple-ratings instead.

Reply:
Forgive me for not understanding,

They use the notation below in the .md files

### Ferramentas recomendadas {#tools} - pt-Br
### Recommended Tools {#tools} - en-US

That way they keep the same reference to #tools. I believe that the documentation here would need to follow the same logic or something similar.

Reply:
Looking at your answer I think that you understood correctly what I had in mind :). I wouldn't worry at the implementation side, it's relatively straightforward to make happen. Now, looking at what React did, this sounds like something we want :).

Reply:
So should we open a new issue?

Reply:
We could, yet :)

Reply:
Opened #20829 :)